### DISCUSSION TITLE
What's the feature allowing partially rendered incorrect equations called?

### DISCUSSION THREAD

**Discussion by rdong8** [KaTeX/KaTeX#3969] [Q&A] [answered]
For instance, in Obsidian if you're in the middle of typing a command (ie. you've typed the `\fra` part of `\frac ab`), you get this:

![image](https://github.com/user-attachments/assets/f164f010-78a6-4364-afe4-69b984532b66)

I want to open a feature request with a piece of software that uses KaTeX for them to enable this so I just need to know what it's called.

**Comment by edemaine** [comment, marked as answer]
It's `throwOnError: false` (not the most intuitive name). See https://katex.org/docs/options
